VsViewer/vs_calc/CPT.py

```diff
@@ -202,7 +202,11 @@ def calc_cpt_params(self):
         for i in range(0, len(n)):
             deltan = 1
             # iterate Qtn, Ic and n until convergence
+            iteration_counter = 0
             while deltan >= 0.01:
+                iteration_counter += 1
+                if iteration_counter > 1e6:
```

Comment:
Perhaps you could define `max_iteration=1e6` as a keyword argument for this function?